ca_description/urdf/april_tag.xacro

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" ?>
+<robot xmlns:xacro="http://ros.org/wiki/xacro">
+
+  <xacro:macro name="april_tag">
+
+    <xacro:property name="april_tag_z_offset" value="0.3"/>
+
+    <link name="april_tag_link">
+      <visual>
+        <origin xyz=" 0 0 ${april_tag_z_offset}" rpy="0 0 0" />
```

Comment:
Make the whole pose customizable as a Xacro parameter.

Reply:
In fact, the origin could be applied to the joint.

Reply:
What difference would it make to make it applied to the joint?

Reply:
The joint "belongs" to the aprilTag. So, when you plot the joints in Gazebo, you will see both in different positions. It's easier in terms of debugging something.